src/components/EpOsaamismerkki/EpOsaamismerkkiModal.vue

```diff
@@ -295,6 +304,10 @@ export default class EpOsaamismerkkiModal extends Vue {
   get muokkausText() {
     return ' - ' + this.$t('muokannut-viimeksi') + ': ' + this.osaamismerkki.muokkaaja + ' ';
   }
+
+  get koodi() {
+    return this.osaamismerkki.koodiUri ? this.osaamismerkki.koodiUri.replace('osaamismerkit_', '') : null;
```

Comment:
arvon voi esittää myös .split('_')[1]
tällöin ei ole riiippuvainen koodiston nimestä.